programs/clearing_house/src/lib.rs

```diff
@@ -989,6 +998,29 @@ pub mod clearing_house {
         Ok(())
     }
 
+    #[access_control(
+        exchange_not_paused(&ctx.accounts.state)
+    )]
+    pub fn update_amms(ctx: Context<UpdateAMM>, market_indexes: [u64; 5]) -> Result<()> {
```

Comment:
not important for now but would be nice to make this market_indexes: [Option<u64>; 5]

Reply:
left a todo